packages/main/src/Button.ts

```diff
@@ -26,7 +26,6 @@ import ButtonType from "./types/ButtonType.js";
 import ButtonAccessibleRole from "./types/ButtonAccessibleRole.js";
```

Comment:
Seems like the `ButtonAccessibleRole` enum is no longer needed. We could use the `AriaRole` enum instead. Same applies for the ui5-link component and the corresponding `accessibleRole` property. There is the following related PR: https://github.com/SAP/ui5-webcomponents/pull/8807/files